main.tex

```diff
@@ -147,15 +143,15 @@ \section*{Introduction}
 %%
 
 %%
-Here we report an in-depth analysis of the VGSC gene, using whole-genome Illumina sequence data from phase 1 of the \emph{Anopheles gambiae} 1000 Genomes Project (Ag1000G) \cite{Ag1000gConsortium2017}.
+Here we report an in-depth analysis of the \textit{Vgsc} gene, using whole-genome Illumina sequence data from phase 1 of the \emph{Anopheles gambiae} 1000 Genomes Project (Ag1000G) \cite{Ag1000gConsortium2017}.
```

Comment:
I believe both "the VGSC gene" and "the _Vgsc_ gene" are valid but picked the latter as something to apply consistently throughout.

Reply:
I was just thinking the same thing! I prefer the latter.